server/routes/calendar.js

```diff
@@ -1,6 +1,42 @@
 var express = require("express");
 var router = express.Router();
 
+var savedAvails = [];
+router.post("/save", (req, res) => {
+  items = req.body.items;
+  console.log(items);
+  savedAvails = items;
+  return res.json({ schedule: items });
+});
+
+router.get("/availability", function(req, res) {
+  pool.query("SELECT * FROM AVAILABILITY", (error, result) => {
```

Comment:
This is unnecessary right? We're not doing anything with the information from the availability table are we?